src/utils.ts

```diff
@@ -46,10 +49,15 @@ export function formatAddress(
   if (options?.withPostalCode) {
     postalCode = `${address.postalCode} `;
   }
+
+  let addressApartmentStr = '';
+  if (options?.addressApartment && options.addressApartment.length > 0) {
```

Comment:
No need to check for length since an empty string would evaluate to false anyway.
```suggestion
  if (options?.addressApartment) {
```